src/main/java/org/bihealth/mi/easybus/implementations/email/BusEmail.java

```diff
@@ -41,22 +43,22 @@ public class BusEmail extends Bus {
     protected abstract static class BusEmailMessage {
         
         /** Receiver */
-        protected final Participant receiver;
+        protected final Participant     receiver;
         /** Scope */
-        protected final Scope       scope;
+        protected final Scope           scope;
         /** Message */
-        protected final Message     message;
+        protected final MessageFragment message;
         /** Subject */
-        protected final String      subject;
+        protected final String          subject;
         
         
         /**
          * Message
          * @param receiver
          * @param scope
-         * @param message
+         * @param attachment
          */
-        BusEmailMessage(Participant receiver, Scope scope, Message message, String subject) {
+        BusEmailMessage(Participant receiver, Scope scope, MessageFragment message, String subject) {
```

Comment:
Ist es worlöoch notwendig, hier von Message -> MessageFragment zu ändern in den Parametern. Ich glaube nicht (MessageFragment ist doch auch eine Message).